data/strings/types_strings.txt

```diff
@@ -18752,7 +18752,7 @@
     nl = Toeristische informatie
     fi = Turisti-informaatio
     fr = Informations touristiques
-    de = Tourist-Information
+    de = Wegweiser
```

Comment:
Ich schlage hier "Tou­rist­in­for­ma­ti­on" vor. 
Wieso "Wegweiser"?

Reply:
Ich habe es so gewählt, wie es im [OSM-Wiki](https://wiki.openstreetmap.org/wiki/DE:Tag:information%3Dguidepost) steht. Bei "Tou­rist­in­for­ma­ti­on" würde ich ein Büro o.ä. erwarten und kein Schild mit Richtungsangaben.

Reply:
Ach so, dann gut.
Wir muessen das natuerlich in allen Sprachen aendern

Reply:
Stimmt, aber das gehört dann nicht hierher.